R/submit-models.R

```diff
@@ -122,8 +123,15 @@ submit_nonmem_models <- function(.mods,
 
     model_dir <- get_model_working_directory(.run[["models"]][[1L]])
 
+    .path_exists <- test_file_exists(
```

Comment:
One last comment (follow-up on https://github.com/metrumresearchgroup/bbr/pull/496/files#r900510130): I see you adjusted the other site to use `file_exists` (thanks for that), but it looks like you forgot to update this site.

Aside from that inconsistency, this looks ready to go to me.

Reply:
Ah yes, got distracted by the drone errors and forgot to make that change. Newest commit reflects that